bus-mapping/src/evm/opcodes/returndatacopy.rs

```diff
@@ -44,6 +62,7 @@ impl Opcode for Returndatacopy {
                     let mut return_data = return_data[data_starts..].to_vec();
                     return_data.resize(data_ends - data_starts, 0);
                     memory[mem_starts..mem_ends].copy_from_slice(&return_data);
+                    // TODO: figure out what should go here?
```

Comment:
precompile.... I remember if we call a non-fixed-len precompile(like identity?) which writes its return data into context's returndata and caller's memory, then we cannot know length of the return data..

@lightsing will make a PR dealing with precompile soon.